src/modules/admin/sections/Appearance/Appearance.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { VStack } from '@chakra-ui/react';
+import AppearanceProfile from './Appearance.Profile';
+
+export default function Appearance() {
+  return (
+    <VStack flex={2} borderRight="1px solid #E7E8EE" pt="80px" spacing={25}>
```

Comment:
This feels better if we have it in the page file. `ProfileEditor` (_or a better name if you feel like it_) can be its own section in the `admin` folder (_at least for now_).

Reply:
Fixed: refactored file structure